pkg/operator/ceph/object/controller.go

```diff
@@ -101,6 +101,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return err
 	}
+	logger.Info("successfully started")
```

Comment:
will the controller ever not start? should these messages in the controllers be debug-level? they're not verbose, just wondering if they're at risk of not starting.

Reply:
If it does not start we will know with an error, I just think this one is nicely informative and is not too verbose.